django-verdant/rca/api/filters.py

```diff
@@ -41,7 +41,7 @@ def filter_queryset(self, request, queryset, view):
             date_from = request.query_params.get('event_date_from', None)
             if date_from:
                 now = datetime.utcnow().date()
-                queryset = queryset.filter(dates_times__date_from__gte=now)
+                queryset = queryset.filter(dates_times__date_from__gte=now) | queryset.filter(dates_times__date_to__gte=now)
```

Comment:
`queryset = queryset.filter(dates_times__date_from__gte=now) | queryset.filter(dates_times__date_to__gte=now)`

Could this be re-written using django.db.models.Q?

`queryset = queryset.filter(Q(dates_times__date_from__gte=now) | Q(dates_times__date_to__gte=now))`